Src/FluentAssertions.Json/JTokenAssertions.cs

```diff
@@ -437,15 +437,17 @@ public AndConstraint<JTokenAssertions> HaveCount(int expected, string because =
         /// <example>
         /// This example asserts the values of multiple properties of a child object within a JSON document.
         /// <code>
-        /// var json = JToken.Parse("{ success: true, data: { id: 123, type: 'my-type', name: 'Noone' } }");
-        /// json.Should().ContainSubtree(JToken.Parse("{ success: true, data: { type: 'my-type', name: 'Noone' } }"));
+        /// var json = JToken.Parse("{ success: true, data: { id: 123, type: 'my-type', name: 'None' } }");
```

Comment:
nit: `None` is not more correct than `Noone`. (no one/no-one).
If tools think that `Noone` is a a typo , then let's just change it to a real name like John or Joe.